### PR TITLE
Modified Role.yml and serviceaccount.yml

### PR DESCRIPTION
**Pull request by ashleshshrivastava**
Modified Role.yml and serviceaccount.yml files for achieving sydbank deployment daily restart usecase

### CODE REVIEW

File: helm/axetrading-cronjob/templates/role.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: Role
 metadata:
   name: {{ include "axetrading-cronjob.fullname" . }}
   labels:
-  {{- include "axetrading-cronjob.labels" . | nindent 4 }}
+    {{- include "axetrading-cronjob.labels" . | nindent 4 }}
```
Comment (viveksinghtt) [inline helm/axetrading-cronjob/templates/role.yaml:6]: Is this intentional

File: helm/axetrading-cronjob/templates/serviceaccount.yaml [hunk 2]
```diff
@@ -10,3 +10,4 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 {{- end }}
+  namespace: '{{ .Release.Namespace }}'
```
Comment (viveksinghtt) [inline helm/axetrading-cronjob/templates/serviceaccount.yaml:13]: Guess this is not needed..

File: helm/axetrading-cronjob/templates/role.yaml [hunk 3]
```diff
@@ -13,3 +13,14 @@ rules:
   - read
   - list
   - get
+- apiGroups:
+  - apps
+  - extensions
```
Comment (viveksinghtt) [inline helm/axetrading-cronjob/templates/role.yaml:18]: We can test and if required add extensions.We can go from minimum privilege to what is required.

**Review by viveksinghtt — APPROVED** [review]
Looks Good...minor comments

**Comment by ursuciprian** [comment]
This PR is included in version 1.0.0 :tada:

**Pull request merged**